packages/perspective-viewer-d3fc/src/js/tooltip/selectionData.js

```diff
@@ -46,7 +46,7 @@ export function getDataValues(data, settings) {
     return [
         {
             name: settings.mainValues[0].name,
-            value: toValue(settings.mainValues[0].type, data.colorValue || data.mainValue - data.baseValue || data.mainValue)
+            value: toValue(settings.mainValues[0].type, data.colorValue || data.mainValue - data.baseValue || data.mainValue || data.mainValues)

```

Comment:
This line's getting a bit confusing, but it'll do for now.

Reply:
I tried to change it in the sunburst code but it didn't seem to work, I'll have another go before I merge.